src/openssh.patch/series

```diff
@@ -1,5 +1,3 @@
 0001-Put-style-as-line-number-to-ssh-session-environment-.patch
 0002-Revert-commit-69334996-make-sshd_config-ClientAliveC.patch
 0003-Export-remote-info-for-authorization.patch
-openssh-7.7p1-fips-8.4p1-5.patch
-microsoft-symcrypt-fips.patch
```

Comment:
Do you need an updated version of this for the provider?

Reply:
Restore the microsoft-symcrypt-fips.patch.
The openssh patch not for 9.2, removed.
If there is a fips tag for 9.2, will add it later.